core/google/cloud/_helpers.py

```diff
@@ -256,9 +258,15 @@ def _time_from_iso8601_time_naive(value):
 
     :rtype: :class:`datetime.time`
     :returns: A datetime time object created from the string
-
+    :raises ValueError: if the value does not match a known format.
     """
-    return datetime.datetime.strptime(value, '%H:%M:%S').time()
+    if len(value) == 8:  # HH:MM:SS
```

Comment:
Similar mild question as #5753 ... should these be regexes instead of length checks?

Reply:
I think not:  `datetime.datetime.strptime` will raise a `ValueError` if the value is bogus:  I'm just using the length to select which format string applies.